games-emulation/bsnes-jg/bsnes-jg-1.1.0.ebuild

```diff
@@ -0,0 +1,46 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit toolchain-funcs
+
+MY_PN=${PN%-*}
+MY_P=${MY_PN}-${PV}
+DESCRIPTION="Jolly Good Fork of bsnes"
+HOMEPAGE="https://gitlab.com/jgemu/bsnes"
+if [[ "${PV}" == *9999 ]] ; then
+	inherit git-r3
+	EGIT_REPO_URI="https://gitlab.com/jgemu/${MY_PN}.git"
+else
+	SRC_URI="https://gitlab.com/jgemu/${MY_PN}/-/archive/${PV}/${MY_P}.tar.bz2"
+	S="${WORKDIR}/${MY_P}"
+	KEYWORDS="~amd64"
+fi
+
+LICENSE="ISC GPL-3+ LGPL-2.1+ MIT ZLIB"
+SLOT="1"
+
+DEPEND="
+	media-libs/libsamplerate
+"
+RDEPEND="
+	${DEPEND}
+	games-emulation/jgrf:1=
+"
+BDEPEND="
+	media-libs/jg:1=
```

Comment:
Is this becaue it's header-only? If so, it belongs in pure DEPEND, not BDEPEND (cross compilers won't ever inspect BROOT for headers).

Reply:
Thanks for pointing this out, I didn't know and I fixed it now.